MAINTAINERS.md

```diff
@@ -16,4 +17,4 @@
 | --------------- | --------- | ----------- |
 | Yan Zeng         | [zengyan-amazon](https://github.com/zengyan-amazon)   | Amazon      |
 | Bandini Bhopi    | [bandinib-amzn](https://github.com/bandinib-amzn)     | Amazon      |
-| Tianle Huang     | [tianleh](https://github.com/tianleh)                 | Amazon      |
```

Comment:
I've been squinting at this line for a minute but can't quite see the difference. Maybe a newline or whitespace character somewhere?

Reply:
It is a line number change. It looks like that Github diff only shows the line number change the last line.